src/components/CustomResources/CustomResourcesOfType.js

```diff
@@ -58,12 +58,6 @@ export default function CustomResourcesOfType({ crdName }) {
           </Link>
         </DynamicPageComponent.Column>
       </DynamicPageComponent>
-      <CustomResources
```

Comment:
Why this is removed?

Reply:
because I forgot about it with `DynamicPageHeader` refactor and with this CustomResources are rendered two times